concent_api/api-integration-force-payment.py

```diff
@@ -44,266 +41,224 @@ def subtask_results_accepted(timestamp = None, payment_ts = None, task_to_comput
-            subtask_results_accepted_list = [
+            timestamp=timestamp_to_isoformat(current_time),
+            subtask_results_accepted_list=[
                 subtask_results_accepted(
-                    timestamp       = timestamp_to_isoformat(current_time),
-                    payment_ts      = current_time,
-                    task_to_compute = task_to_compute(
-                        timestamp                       = timestamp_to_isoformat(current_time),
-                        requestor_ethereum_public_key   = "x" * 128,
-                        price=0,
-                        compute_task_def                = compute_task_def(
-                            deadline = current_time,
-                            task_id  = task_id + 'a',
-                            subtask_id=task_id + 'A'
-                        )
+                    timestamp=timestamp_to_isoformat(current_time),
+                    payment_ts=current_time,
+                    task_to_compute=create_signed_task_to_compute(
+                        timestamp=parse_timestamp_to_utc_datetime(current_time),
+                        task_id=task_id + 'a',
+                        subtask_id=subtask_id + 'A',
+                        deadline=current_time,
+                        requestor_ethereum_public_key="x" * 128,
+                        price=1000,
                     )
                 ),
                 subtask_results_accepted(
-                    timestamp       = timestamp_to_isoformat(current_time),
-                    payment_ts      = current_time,
-                    task_to_compute = task_to_compute(
-                        timestamp                       = timestamp_to_isoformat(current_time),
-                        requestor_ethereum_public_key   = "x" * 128,
-                        price=0,
-                        compute_task_def                = compute_task_def(
-                            deadline = current_time,
-                            task_id  = task_id + 'b',
-                            subtask_id=task_id + 'B'
-                        )
+                    timestamp=timestamp_to_isoformat(current_time),
+                    payment_ts=current_time,
+                    task_to_compute=create_signed_task_to_compute(
+                        timestamp=parse_timestamp_to_utc_datetime(current_time),
+                        task_id=task_id + 'b',
+                        subtask_id=subtask_id + 'B',
+                        deadline=current_time,
+                        requestor_ethereum_public_key="x" * 128,
+                        price=1000,
                     )
                 )
             ]
         ),
 
-        headers = {
-            'Content-Type':                     'application/octet-stream',
-            'temporary-list-of-transactions':   'True',
-            'temporary-v':                      '-1',
-            'temporary-eth-block':              '-1'
+        headers={
```

Comment:
Would it make sense to make this header default inside `api_request` call ? It is repeated in almost every call.